docs/example_static_html.md

```diff
@@ -3,50 +3,190 @@ title: "Example: Plain Old Static HTML"
-cd tilt/integration/oneup
+
+To start this server on Kubernetes, we need 3 configs:
+
+1) A [Dockerfile](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/Dockerfile) that builds the image.
+
+2) A [Kubernetes deployment](https://github.com/windmilleng/tilt-example-html/blob/master/0-base/kubernetes.yaml) that runs the image
+
+3) And finally, a Tiltfile that ties them together.
+
+```python
+docker_build('example-html-image', '.')
+k8s_yaml('kubernetes.yaml')
+k8s_resource('example-html', port_forwards=8000)
 ```
 
-We'll be looking at the [oneup](https://github.com/windmilleng/tilt/tree/master/integration/oneup) project
-in `integration/oneup`.
+The first line tells Tilt to build an image with the name `example-html-image`
+in the directory `.` (the current directory).
 
-In the oneup directory, run
+The second line tells Tilt to load the Kubernetes deployment yaml. The image name in the `docker_build`
+call must match the image name in the `example-html` Deployment.
+
+The last line configures port-forwarding so that your server is
+reachable at http://localhost:8000/. The resource name in the `k8s_resource` call
+must match the deployment name in the Kubernetes yaml.
+
+Try it! Run:
 
 ```
+git clone https://github.com/windmilleng/tilt-example-html
+cd tilt-example-html/0-base
 tilt up
 ```
 
-Your terminal will turn into a status box that lets you watch your server come up. When it's ready,
-you will see the status icon turn green. The logs in the botton pane will display
-"Serving oneup on container port 8000."
+Your terminal will turn into a status box that lets you watch your server come
+up. When it's ready, you will see the status icon turn green. The logs in the
+botton pane will display "Serving files on port 8000." The snapshot below
+lets you see what it should look like.
+
+[Snapshot](https://cloud.tilt.dev/snapshot/AejkyuULr2AjWu50Eck=)
+
+## Step 1: Let's Add Benchmark Trickery
+
+Before we try to make this faster, let's measure it.
+
+Tilt has two major escape hatches for running commands locally:
+
+- `local(command)` which runs a local command for its output
+
+- `local_resource(name, command)` which lets you run local jobs to trigger new updates, or run local servers
+
+For more detail, see the [local resource guide](local_resource.html)
+
+We add a `local_resource` to our
+[Tiltfile](https://github.com/windmilleng/tilt-example-html/blob/master/1-measured/Tiltfile)
+that records when an update starts.
+
+```python
+k8s_resource('example-html', port_forwards=8000, resource_deps=['deploy'])
+
+# Records the start time.
+# When the contents of this directory change, example-html will automatically re-deploy.
+local_resource(
+  'deploy',
+  'date +%s > start-time.txt',
+  trigger_mode=TRIGGER_MODE_MANUAL)
+```
+
+The `local_resource()` call creates a local resource named `deploy`. The second
+argument is the script that it runs. The third argument adds a button to the
+Tilt UI, to re-start the deploy.
```

Comment:
why is `TRIGGER_MODE_MANUAL` necessary here? it has no deps, it would never auto-run

Reply:
good point!